pkgs/by-name/pa/passhole/package.nix

```diff
@@ -0,0 +1,60 @@
+{
+  lib,
+  fetchPypi,
+  python3,
+}:
+let
+  pname = "passhole";
+  version = "1.10.0";
+
+  pykeepass-cache = python3.pkgs.buildPythonPackage {
+    pname = "pykeepass-cache";
+    version = "2.0.3";
+
+    src = fetchPypi {
+      pname = "pykeepass-cache";
+      version = "2.0.3";
+      sha256 = "sha256-fzb+qC8dACPr+V31DV50ElHzIePdXMX6TtepTY6fYeg=";
+    };
+
+    dependencies = with python3.pkgs; [
+      pykeepass
+      rpyc
+      python-daemon
+    ];
+  };
+in
+python3.pkgs.buildPythonPackage {
+  inherit pname version;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-3pNxhqmkw8tO0wVBuZnua64HJrEpyeNSAKiAMsM4oVY=";
+  };
+
+  dependencies =
+    with python3.pkgs;
+    [
+      pynput
+      pykeepass
+      colorama
+      future
+      pyotp
+      qrcode
+    ]
+    ++ [ pykeepass-cache ];
+
+  checkPhase = ''
+    ${python3}/bin/python test/tests.py
+  '';
+
+  meta = {
+    description = "A secure hole for your passwords (KeePass CLI)";
```

Comment:
```suggestion
    description = "KeePass CLI inspired by pass";
```

Reply:
I went by the description provided by the official repo (https://github.com/Evidlo/passhole). Just copied their "About" section.

Reply:
I basically copied the first line of the readme. You need to at least remove the leading "A " to comply with nixpkgs requirements.

Reply:
Dropped the A. 739b5e7396157308254f6f7680b755744bd9867e